explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -417,82 +391,13 @@ private object SpectroscopyModesTable extends ModesTableCommon:
                             else Callback.empty
         visibleRows    <- useState(none[Range.Inclusive]) // visibleRows
         atTop          <- useState(false)                 // atTop
-        // Recalculate ITC values if the wv or sn change or if the rows get modified
-        itcValues      <-
-          useEffectStreamResourceWithDeps(
-            (props.spectroscopyRequirements.exposureTimeMode,
-             props.constraints,
-             props.validTargets,
-             props.customSedTimestamps,
-             rows.length
-            )
-          ): (expTimeMode, constraints, asterism, customSedTimestamps, _) =>
-            import ctx.given
-
-            (expTimeMode, expTimeMode.map(ExposureTimeMode.at.get), asterism)
-              .mapN: (expTimeMode, snAt, asterism) =>
-                val modes: List[SpectroscopyModeRowWithResult] =
-                  sortedRows
-                    .filterNot: row => // Discard modes already in the cache
-                      val cache: Map[ItcRequestParams, EitherNec[ItcTargetProblem, ItcResult]] =
-                        itcResults.value.cache
-
-                      // center of the row range
-                      val cw: Option[CentralWavelength] =
```

Comment:
The check was removed when extracting the logic, because rows that don't have a central wavelength are already filtered out of the table above.